tests/test_regrid.py

```diff
@@ -212,19 +199,9 @@ def test_conservative_nan_thresholds_against_coarsen(nan_threshold):
     xr.testing.assert_allclose(da_coarsen, da_regrid)
 
 
-def xesmf_available() -> bool:
-    try:
-        import xesmf  # noqa: F401
-    except ImportError:
-        return False
-    return True
-
-
-@pytest.mark.skipif(not xesmf_available(), reason="xesmf required")
+@pytest.mark.skipif(xesmf is None, reason="xesmf required")
 def test_conservative_nan_thresholds_against_xesmf():
-    import xesmf as xe
-
-    ds = xr.tutorial.open_dataset("ersstv5").sst.compute()
+    ds = xr.tutorial.open_dataset("ersstv5").sst.compute().isel(time=[0])
```

Comment:
```suggestion
    ds = xr.tutorial.open_dataset("ersstv5")["sst"].isel(time=[0]).compute()
```
First selecting and then computing should be a little more efficient right? But looking at the dataset's size it probably won't matter much.

Reply:
This was the weird bug with current version of `netcdf4`. It adds very little time but yeah hopefully they fix it or we could switch to `h5netcdf` in the env. I'll leave it as is for now.